src/Microsoft.Identity.Web.Sidecar/Dockerfile.linux

```diff
@@ -0,0 +1,27 @@
+# syntax=docker/dockerfile:1.7
+ARG DOTNET_VERSION=9.0
+ARG SDK_IMAGE=mcr.microsoft.com/dotnet/sdk:${DOTNET_VERSION}-alpine
+ARG RUNTIME_IMAGE=mcr.microsoft.com/dotnet/aspnet:${DOTNET_VERSION}-alpine
+
+FROM ${SDK_IMAGE} AS restore
+WORKDIR /src
+COPY src/Microsoft.Identity.Web.Sidecar/Microsoft.Identity.Web.Sidecar.csproj src/Microsoft.Identity.Web.Sidecar/
+COPY src/Microsoft.Identity.Web/Microsoft.Identity.Web.csproj src/Microsoft.Identity.Web/
+COPY src/Microsoft.Identity.Web.AgentIdentities/Microsoft.Identity.Web.AgentIdentities.csproj src/Microsoft.Identity.Web.AgentIdentities/
+COPY src/Microsoft.Identity.Web.DownstreamApi/Microsoft.Identity.Web.DownstreamApi.csproj src/Microsoft.Identity.Web.DownstreamApi/
+RUN dotnet restore src/Microsoft.Identity.Web.Sidecar/Microsoft.Identity.Web.Sidecar.csproj --use-current-runtime
+
+FROM restore AS build
+COPY . .
+RUN dotnet publish src/Microsoft.Identity.Web.Sidecar/Microsoft.Identity.Web.Sidecar.csproj -c Release -o /app/publish --no-restore
+
+FROM ${RUNTIME_IMAGE} AS runtime
+WORKDIR /app
+RUN addgroup -S app && adduser -S app -G app
```

Comment:
```suggestion
```

Since net8 containers, the 'app' user is created in the lower layer:
https://devblogs.microsoft.com/dotnet/securing-containers-with-rootless/